lib/profile/node.rb

```diff
@@ -43,7 +44,8 @@ def self.list_hunter_nodes
         parts = line.split("\t").map { |p| p.empty? ? nil : p }
         new(
           hostname: parts[1],
-          hunter_label: parts[4]
+          hunter_label: parts[4],
+          ip: parts[2]
```

Comment:
Is the IP attribute needed/going to be used when listing nodes for Flight Hunter?

Reply:
I'm not sure what you mean by this - the node's IP is needed for the new Ansible inventory format, and it's obtained from the IP collected by Hunter.

Reply:
My mistake; I misread this as being something else.